indico/web/client/styles/base/_typography.scss

```diff
@@ -62,35 +66,36 @@ body {
     background: #fff;
     padding: 0px;
     margin: 0px;
-}
-
-body > :not(.rb-app-container) {
```

Comment:
It makes sense to read this and understand that the "Liberation Sans" font styles are applied except for the room booking container. However, I couldn't find any evidence of this CSS selector (.rb-app-container) ever being used. That's why it has been removed.

Reply:
i don't think we include anything from the "base" css in room booking anyway

Reply:
I don't think this is supposed to be removed. There are cases where no other CSS rules from `_typography.scss` (like `.conference-page` or `.description` etc) are applied and then you need this rule to ensure that "Liberation Sans" and not "Muli" is applied.

Reply:
But the behavior doesn't change as far as I can see, since `:not(.rb-app-container)` was always true.

The only problem I could see is that `body > xxx` is more specific than `body`, so in some cases it may not apply this anymore now.

Reply:
But since we have included SUI everywhere in indico, it's trying to enforce a different font in the body and we need this rule in order to avoid that.

Reply:
Ah, I understand now. My bad. Thought he meant that the entire rule was not needed.